devicetypes/smartthings/zwave-lock-without-codes.src/zwave-lock-without-codes.groovy

```diff
@@ -405,6 +408,11 @@ private def handleBatteryAlarmReport(cmd) {
 	def deviceName = device.displayName
 	def map = null
 	switch (cmd.zwaveAlarmEvent) {
+		case 0x01: //power has been applied, check if the battery level updated
+			state.queryBattery = true
```

Comment:
Seems like this DTH needs something that sets `queryBattery` to `false` similar to the other DTH

Reply:
yep. thanks